### ISSUE TITLE
[Bug] Can't change UI language via settings

### ISSUE THREAD

**Issue by tryallthethings** [hluk/CopyQ#1562]
**Describe the bug**
Can't change UI language via settings

**To Reproduce**
1. Go to settings
2. Change language from current language (German) to another language (English in my case)
3. Click apply
4. Restart CopyQ

**Expected behavior**
Steps should change the language, but the UI language stays German even after restart of app.

**Screenshots**

**Version, OS and Environment**
 - Application Version: Latest (3.13.0)
 - OS: Windows 8.1

**Additional context**
This is a fresh installation of CopyQ - no other settings were made. For any other language but English a popup is shown stating that I need to restart the application to apply the language change. But the lanuage stays German regardless of which one I pick.

**Comment by hluk** [comment]
Is there any error or warning in logs (F12 shortcut from main window)?

Also check if there is `language=en` in the configuration file `copyq.ini`. See: [Where to find saved items and configuration?](https://copyq.readthedocs.io/en/latest/faq.html#where-to-find-saved-items-and-configuration)

**Comment by tryallthethings** [comment]
There was no config file created in %APPDATA%\copyq

Log:
```
ERROR [2021-01-13 23:23:40.554] <Server-3452>: Failed to save commands in new file C:/Users/root.s/AppData/Roaming/copyq/copyq-commands.ini
ERROR [2021-01-13 23:27:56.529] <Server-6736>: Failed to save commands in new file C:/Users/root.s/AppData/Roaming/copyq/copyq-commands.ini
ERROR [2021-01-13 23:30:50.725] <Server-7728>: Failed to save commands in new file C:/Users/root.s/AppData/Roaming/copyq/copyq-commands.ini
ERROR [2021-01-13 23:35:08.112] <Server-13444>: Failed to save commands in new file C:/Users/root.s/AppData/Roaming/copyq/copyq-commands.ini
ERROR [2021-01-13 23:38:08.585] <Server-13756>: Failed to save commands in new file C:/Users/root.s/AppData/Roaming/copyq/copyq-commands.ini
ERROR [2021-01-13 23:42:33.412] <Server-10496>: Failed to save commands in new file C:/Users/root.s/AppData/Roaming/copyq/copyq-commands.ini
ERROR [2021-01-22 02:20:55.227] <Server-5100>: Failed to save commands in new file C:/Users/root.s/AppData/Roaming/copyq/copyq-commands.ini
```

My user has administrative permissions as well as read / write permissions in that folder.
Content of the folder:
![image](https://user-images.githubusercontent.com/11146296/105433187-746bd200-5c59-11eb-9ce0-1a2408e18104.png)

Now that I see where the issue might be coming from I've tried running CopyQ as Administrator and a bunch of files were created:

![image](https://user-images.githubusercontent.com/11146296/105433345-b7c64080-5c59-11eb-83be-f0a515966033.png)
No new log entries were generated when running it as Administrator.

**Comment by hluk** [comment]
Yeah, running the app as admin is not a good idea. I'm not sure how to detect it in the app. Maybe we can have an FAQ entry in the docs at least.

Can you change permissions of the files so that non-admin user has write permission?

**Comment by tryallthethings** [comment]
The folder permissions are the same as with any other folder in %APPDATA%:
![image](https://user-images.githubusercontent.com/11146296/106833223-768b5300-6693-11eb-8927-4699932d8ed9.png)

And my user is part of the Local Admin group, so he should have enough permissions anyway. The 2 subfolders and the .dat files could be created. Only the other .ini files are only created if the CopyQ is started with "Run as Administrator"

**Comment by Chealer** [comment]
Does this persist with version 13 or ulterior? If so, how was CopyQ installed?
